Runtime/Data/Type/TransformData.cs

```diff
@@ -3,82 +3,77 @@
         /// </summary>
-        [DocumentedByXml]
-        public Transform transform;
+        [Serialized]
+        [field: DocumentedByXml]
+        public Transform Transform { get; set; }
         /// <summary>
-        /// Position override of the <see cref="Transform"/> object.
+        /// Position override of the <see cref="UnityEngine.Transform"/> object.
         /// </summary>
-        [DocumentedByXml]
-        public Vector3? positionOverride;
+        [Serialized]
+        [field: DocumentedByXml]
+        public Vector3? PositionOverride { get; set; }
         /// <summary>
-        /// Rotation override of the <see cref="Transform"/> object.
+        /// Rotation override of the <see cref="UnityEngine.Transform"/> object.
         /// </summary>
-        [DocumentedByXml]
-        public Quaternion? rotationOverride;
+        [Serialized]
+        [field: DocumentedByXml]
+        public Quaternion? RotationOverride { get; set; }
         /// <summary>
-        /// Scale override of the <see cref="Transform"/> object.
+        /// Scale override of the <see cref="UnityEngine.Transform"/> object.
         /// </summary>
-        [DocumentedByXml]
-        public Vector3? scaleOverride;
+        [Serialized]
+        [field: DocumentedByXml]
+        public Vector3? ScaleOverride { get; set; }
 
         /// <summary>
-        /// The position of the <see cref="Transform"/> or the <see cref="positionOverride"/> if it is set.
+        /// The position of the <see cref="UnityEngine.Transform"/> or the <see cref="PositionOverride"/> if it is set.
         /// </summary>
-        public Vector3 Position => positionOverride ?? transform.position;
+        public Vector3 Position => PositionOverride ?? Transform.position;
 
         /// <summary>
-        /// The rotation of the <see cref="Transform"/> or the <see cref="rotationOverride"/> if it is set.
+        /// The rotation of the <see cref="UnityEngine.Transform"/> or the <see cref="RotationOverride"/> if it is set.
         /// </summary>
-        public Quaternion Rotation => rotationOverride ?? transform.rotation;
+        public Quaternion Rotation => RotationOverride ?? Transform.rotation;
 
         /// <summary>
-        /// The scale of the <see cref="Transform"/> or the <see cref="scaleOverride"/> if it is set.
+        /// The scale of the <see cref="UnityEngine.Transform"/> or the <see cref="ScaleOverride"/> if it is set.
         /// </summary>
-        public Vector3 Scale => scaleOverride ?? transform.lossyScale;
+        public Vector3 Scale => ScaleOverride ?? Transform.lossyScale;
 
         /// <summary>
         /// The state of whether the <see cref="TransformData"/> is valid.
         /// </summary>
-        public bool Valid
-        {
-            get
-            {
-                return (transform != null);
-            }
-        }
+        public bool Valid => Transform != null;

```

Comment:
```suggestion
        public bool IsValid => Transform != null;
```
... while we're here.